bddframework/pkg/api/app/v1beta1/groupversion_info.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package v1beta1 contains API Schema definitions for the app v1beta1 API group
+// +kubebuilder:object:generate=true
+// +groupName=app.kiegroup.org
+package v1beta1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "app.kiegroup.org", Version: "v1beta1"}
```

Comment:
These types were migrated to keep compatibility with the previous work, but are not used here, right? This version follows the Kogito Operator.

Reply:
Well, my initial thought was that if in future we are going to use some kind of supporting services like we used to in Kogito (be it Data Index, UI or whatever), this may be a good starting point in tests to reuse those steps and just repurpose these structs. But yes, those are project-specific, so before extracting the whole framework to a separate repository we could first move them under the `testbdd` module or simply remove them.